ansible/README.md

```diff
@@ -14,23 +14,23 @@ can be real hardware, VMs, things in a public cloud, etc. Anything that you can
 
 ### Configure inventory
 
-Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster.
+Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster. Place the 'inventory' file into the `./inventory` directory.
 
 ### Configure Cluster options
 
-Look though all of the options in `group_vars/all.yml` and
+Look though all of the options in `inventory/group_vars/all.yml` and
 set the variables to reflect your needs. The options are described there
 in full detail.
 
 ## Running the playbook
 
-After going through the setup, run the setup script provided:
+After going through the setup, run the deploy-cluster script provided by going into the scripts directory:
```

Comment:
One can ``deploy`` etcd cluster only, or just nodes. So we decide to rename so it is aligned with entire cluster deployment use case.